docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -408,70 +437,79 @@ over. This ends the turn order, and characters can act independently of turn ord
+  the relevant sense.
+
+  5. **Knocked Out** Knocked Out (KO) characters are unable to act until treated by a Craft
+  check or a Healing Power (DR 5 vs. the KO character's TC, minus 1 for each turn passed
+  since the effect onset). If a Knocked Out Dealer is attacked further, the Player begins
+  an Epic Event alone to avoid *Death*.
+
+  6. **Burned** As a Minor Action, a character can make a DR 3 GUT Save to stop the
+  burning. On failure, they take 1 damage bypassing AP.
+
+  7. **Poisoned N** Each poisoning effect adds +1 Poisoned. At the start of their turn,
+  as a Free Action, a character makes a DR 3 VIT Save to shake off the effects of the
+  poison. On failure, they discard N+1 cards.
+
+  8. **Frozen** Movement speed is reduced by half and all Strength and Agility based
+  skill checks are drawn at Lower Hand. A character makes a DR 3 Strength Save as a Free
+  Action at the end of their turn to attempt to end the freeze.
+
+  9. **Suffocating** At the beginning of their turn, a character makes a DR 3 VIT Save.
+  On failure, take 2 damage bypassing AP. On a success, take 1 damage.
+
+  10. **Charmed** At the start of their turn or whenever taking damage, a character
+  makes a DR 3 GUT Save. On failure, they may not take actions against the interests of
+  the creature that caused the condition. Another creature within 6 spaces may use a
+  Major action to end this effect. This effect ends automatically if the target is
+  damaged by the creature that caused the condition.
+
+  11. **Enthralled** At the start of their turn or whenever taking damage, a character
+  makes a DR 3 CON Save. On failure, they must move up to their Speed and/or perform 1
+  Major Action available to them, as directed by the creature that caused the condition.
+  Another creature within 6 spaces may use a Major action to end this effect. This
+  effect ends automatically if the target is damaged by the creature that caused the
+  condition.
+
+  12. **Invisible** When making a Check to detect an Invisible creature, draw with Lower
+  Hand. Unless otherwise stated, using a Power ends the Invisible condition. The attack
+  that ends Invisibility is drawn at Upper Hand.
+
+  13. **Grappled** Movement speed is reduced to 0 and a Grappled creature cannot gain
+  the benefit of additional speed. They have Lower Hand when using any Power. A
+  creature can end the Grappled condition by making a successful Contested Brute or
+  Finesse Check against target's Brute.
+
+  14. **Dead** If a Knocked Out Dealer is attacked further, they begin an Epic Event. On
+  failing the Epic Event, the Dealer is dead in the game world. They live on in the
+  Player's memory and the other characters of the world. Players are encouraged to
+  create a new Dealer and work with their GM to introduce that character in a future
+  session. Some GMs may decide that Player Character *Death* is not part of their story.
```

Comment:
This forces mechanics where I would rather DM choice. What about an adventure to retrieve a party member from the great beyond?

Reply:
Will remove for now, and potentially re-add as a section in the GM's guide for optional ways to handle Death.